samples/zoomable/src/main/java/com/facebook/samples/zoomable/ZoomableDraweeView.java

```diff
@@ -115,6 +116,33 @@ private void init() {
     mZoomableController = createZoomableController();
     mZoomableController.setListener(mZoomableListener);
     mTapGestureDetector = new GestureDetector(getContext(), mTapListenerWrapper);
+    mTapGestureDetector.setOnDoubleTapListener(new GestureDetector.OnDoubleTapListener() {
+      @Override
+      public boolean onSingleTapConfirmed(MotionEvent e) {
+        return false;
+      }
+
+      @Override
+      public boolean onDoubleTap(MotionEvent e) {
+        float maxScaleFactor = ((DefaultZoomableController) mZoomableController).getMaxScaleFactor();
```

Comment:
The controller is not necessarily a `.DefaultZoomableController` in which case this would crash.